src/server/api/routers/event.ts

```diff
@@ -0,0 +1,54 @@
+import { eq, gte, lte, and } from 'drizzle-orm';
+import { createTRPCRouter, publicProcedure } from '../trpc';
+import { z } from 'zod';
+import { selectEvent } from '@src/server/db/models';
+
+const byClubIdSchema = z.object({
+  clubId: z.string().default(''),
+  startTime: z.date(),
+  endTime: z.date(),
+});
```

Comment:
Do we plan on using the same schema for both for convenience or is this just temporary?